newIDE/app/src/MainFrame/EditorContainers/HomePage/GetStartedSection/RecommendationList.js

```diff
@@ -299,59 +285,6 @@ const RecommendationList = ({
             </SectionRow>
           );
 
-        if (quickCustomizationRecommendation) {
-          items.push(
```

Comment:
Alternative: keep this but sending a version number to the recommendation list and have the backend return things according to it. A bit more work but would keep quick customization in the get started page if needed?

Reply:
Not convinced by this, at the moment the recommendations are pretty fixed.
And from now, we will always display them on the create page, so it feels like a duplicate
(+ we'll rework the get started too)